staging/src/github.com/golang/glog/go.mod

```diff
@@ -0,0 +1,9 @@
+module kubevirt.io/kubevirt/staging/src/github.com/golang/glog
```

Comment:
hm, shouldn't this be `github.com/golang/glog`?

Reply:
good catch! it was autogenerated by `mod init`